### PR TITLE
 DataGrid - Performance drops on adding a large number of rows via editing.changes (T1251098)

### PR DESCRIPTION
**Pull request by tongsonbarbs**

### CODE REVIEW

File: packages/devextreme/js/__internal/grids/grid_core/editing/m_editing.ts [hunk 1]
```diff
@@ -1770,8 +1764,7 @@ class EditingControllerImpl extends modules.ViewController {
     return deferred.promise();
   }
 
-  // @ts-expect-error
-  private _resolveAfterSave(deferred, { cancel, error } = {}) {
+  private _resolveAfterSave(deferred, { cancel = false, error = undefined } = {}) {
```
Comment (pomahtri) [inline packages/devextreme/js/__internal/grids/grid_core/editing/m_editing.ts]:
```suggestion
  private _resolveAfterSave(deferred, { cancel = undefined, error = undefined } = {}) {
```
Minor
Let's not change behavior, looks unrelated to PR

Comment (tongsonbarbs) [reply packages/devextreme/js/__internal/grids/grid_core/editing/m_editing.ts]: done

**Review by pomahtri — APPROVED** [review]
